library/sys.po

```diff
@@ -519,9 +518,9 @@ msgid ""
 msgstr ""
 "Ces objets contiennent les valeurs originales de ``breakpointhook``, "
 "``displayhook`` et ``excepthook`` au début du programme.  Ils sont "
```

Comment:
Il manque ``unraisablehook`` à la liste. La modification upstream qui a changé ce paragraphe est : 

```diff
-   ``displayhook``, and ``excepthook`` at the start of the program.  They are
-   saved so that ``breakpointhook``, ``displayhook`` and ``excepthook`` can be
-   restored in case they happen to get replaced with broken or alternative
-   objects.
+   ``displayhook``, ``excepthook``, and ``unraisablehook`` at the start of the
+   program.  They are saved so that ``breakpointhook``, ``displayhook`` and
+   ``excepthook``, ``unraisablehook`` can be restored in case they happen to
+   get replaced with broken or alternative objects.
```